stores/searchResultsStore.ts

```diff
@@ -38,46 +38,69 @@ export const useSearchResultsStore = defineStore('searchResultsStore', () => {
         //get the facilities that match the professionals we found
         const facilitiesSearchResults = await queryFacilities(professionalIds, selectedCity.value)
 
+        const facilityMap = new Map<string, FacilitySearchResult>()
+
+        facilitiesSearchResults.forEach(facility => {
```

Comment:
you can change this foreach to a map() function that builds the array  of combined objects

Reply:
using map also gives you a variable that will help this be more understandable what the output of this action is.